omero/developers/Java.txt

```diff
@@ -345,50 +345,29 @@ Write data
 -  **Create a dataset and link it to an existing project.**
 
 ::
```

Comment:
`.. code-block:: java` would give you syntax highlighting but maybe that's outwith this PR and better applied in a more global effort.